backend/app/routes/joinUs/index.js

```diff
@@ -1,8 +1,12 @@
 const router = require('express').Router({ mergeParams: true });
 const postJoinUs = require('./post');
+const getJoinUs = require('./get');
 const JoinUsValidationSchema = require('./@validationSchema');
 const validation = require('../../../helpers/middlewares/validation');
+const {authMiddleware} = require('../../../helpers/middlewares/auth')
 
-router.post('/joinUs', validation(JoinUsValidationSchema), postJoinUs);
+router.post('/', validation(JoinUsValidationSchema), postJoinUs);
+// router.get('/', authMiddleware, getJoinUs);
```

Comment:
Why did we removed the authmiddleware?

Reply:
Sorry, I commented it while I'm debugging.